autospec/tarball.py

```diff
@@ -561,6 +570,11 @@ def process(url_arg, name_arg, ver_arg, target, archives_arg, filemanager):
     # name is created by adding ".gcov" to the package name (if a gcov file
     # exists)
     set_gcov()
+    # build the target path, if necessary
+    if not target and package_dir:
+        target = os.path.join(package_dir, name)
+    # create the path, by cloning existing repo or manually, as necessary
+    call("make clone_%s" % name, check=False, stderr=subprocess.DEVNULL)
```

Comment:
I think autospec should also try to clone the repo with git if the `make clone_PKG` command fails. And if that second clone attempt fails, then autospec will create a new git repo locally, which I believe it already does.